### PR TITLE
url check

### PR DESCRIPTION
**Pull request by stojg**
.. and remove a debug dev/task

### CODE REVIEW

File: templates/Layout/DNRoot_alerts.ss [hunk 1]
```diff
@@ -13,18 +13,15 @@
 
 <div class="content">
 
-	<% if $AlertsConfigContent %>
-		<h3>Currently configured alerts</h3>
-		<pre>$AlertsConfigContent</pre>
+	<% if $AlertsConfigContent(master) %>
```
Comment (halkyon) [inline templates/Layout/DNRoot_alerts.ss]: Can you confirm this doesn't double the requests, i.e. that is caches the first call?

File: templates/Layout/DNRoot_alerts.ss [hunk 2]
```diff
@@ -13,18 +13,15 @@
...
 	<% end_if %>
 
 	<h3>Configuring alerts</h3>
 
 	<p>
-		Alerts are sent when a check against an
-		<a href="https://github.com/silverstripe-labs/silverstripe-environmentcheck/">environmentcheck module suite</a>
-		fails with a bad HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in
-		the root of your site code.
+		Alerts are sent when a check HTTP Request against a URL fails with a non 200 HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in the root of your site code.
```
Comment (halkyon) [inline templates/Layout/DNRoot_alerts.ss]: How about this simplified text? "Alerts are sent when a check fails with a non-200 HTTP response status. You configure these checks by placing a <code>.alerts.yml</code> file in the root of your site code."

File: code/service/AlertService.php [hunk 3]
```diff
@@ -29,14 +31,15 @@ public function getAlertsConfigContent($project) {
...
+	 * @param string $sha
 	 * @param DeploynautLogFile $log
+	 * @param DNProject $project
 	 *
-	 * @return boolean
+	 * @return bool
 	 */
-	public function sync($project, $environment, $log) {
-		$content = $this->getAlertsConfigContent($project);
+	public function sync($environment, $sha, $log, $project) {
```
Comment (stojg) [inline code/service/AlertService.php]: The reason I changed order of this is to make it consistent with what the extension is recieving

File: templates/Layout/DNRoot_alerts.ss [hunk 4]
```diff
@@ -79,11 +76,9 @@ alerts:
 	</ul>
 
 	<p>
-		Please contact the <a href="http://helpdesk.silverstripe.com" target="_blank">SilverStripe Operations Team</a>
-		if you would like to add another contact to the list above.
+		For the moment, please raise a <a href="http://helpdesk.silverstripe.com" target="_blank">helpdesk</a> ticket with our operations team and tell us the email addresses that should receive notifications.
```
Comment (halkyon) [inline templates/Layout/DNRoot_alerts.ss]: Probably would say something like: "Please raise a <a href="http://helpdesk.silverstripe.com" target="_blank">helpdesk</a> ticket and advise us which email addresses should receive notifications."

File: templates/Layout/DNRoot_alerts.ss (hunk 1 above)
Comment (stojg) [inline templates/Layout/DNRoot_alerts.ss]: confirmed, it's cached. and it's still a very quick check:

sprintf('git show --format=raw %s:.alerts.yml', $sha);

**Pull request merged**